app/javascript/dashboard/assets/scss/widgets/_modal.scss

```diff
@@ -71,7 +71,7 @@
     @include padding($space-large);
   }
 
-  form {
+  form, .modal-content {
```

Comment:
Please test the rest of the modals as well, since you are changing it globally.

Reply:
This is a new class since all modals were having padding only for form tags, tested other modals as well, worked without issues.